src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -167,4 +171,16 @@ private function getTimezone(array $context): ?\DateTimeZone
 
         return $dateTimeZone instanceof \DateTimeZone ? $dateTimeZone : new \DateTimeZone($dateTimeZone);
     }
+
+    private function forceTimezone(\DateTime|\DateTimeImmutable $object, array $context): \DateTime|\DateTimeImmutable
```

Comment:
why not this?
```suggestion
    private function forceTimezone(\DateTimeInterface $object, array $context): \DateTimeInterface
```

Reply:
Correct that. DateTimeInterface doesn't have the `setTimezone` method. only DateTime and DateTimeImmutable have. I've restored the function argument type. Kept the return type on DateTimeInterface